koku/api/query_params.py

```diff
@@ -406,6 +440,14 @@ def _set_tag_keys(self):
             with tenant_context(self.tenant):
                 self.tag_keys.update(tag_model.objects.values_list("key", flat=True).distinct())
 
+    def _set_aws_category_keys(self):
+        """Set the valid aws_category keys"""
+        self.aws_category_keys = set()
+        # TODO: Right now we accept any key passed in.
+        # However, we should rewrite this to only
+        # allow acceptable categorites.
+        return
```

Comment:
It is not going to completed during this issue, I talked to Andrew this is an MVP and for some internal stakeholders. We are going to build the right urls for a customer and add them to an email we control as a quickest win scenario. Then we will go back and create a table that has the acceptable categories since the categories are created by the user it will work very similarly to what we do for tags in the `reporting_awstags_summary` table. However, that will be in a separate PR, and once we have that we can figure out if a category is acceptable as a parameter. So, these todos are to make it easier for when we come back to it later.